### ISSUE TITLE
PostCSS Warning Displayed on Dev Server's Initial Load.

### ISSUE THREAD

**Issue by yasuhiro-yamamoto** [Dwlad90/stylex-swc-plugin#236]
## Description

When starting the development server, the following warning appears:

```
"Without `from` option PostCSS could generate wrong source map and will not find Browserslist config. 
Set it to CSS file path or to `undefined` to prevent this warning."
```

Although the project functions as expected, this warning is shown only on the initial load of the dev server.

### Steps to Reproduce
```bash
npx create-next-app@latest --use-pnpm
```

```bash
pnpm add @stylexjs/stylex @stylexswc/nextjs-plugin
```

```bash
pnpm add -D @stylexswc/rs-compiler
``` 

#### dependencies / devDependencies
```json
{
  "dependencies": {
    "@stylexjs/stylex": "^0.10.1",
    "@stylexswc/nextjs-plugin": "^0.6.5",
    "next": "15.1.7",
    "react": "^19.0.0",
    "react-dom": "^19.0.0"
  },
  "devDependencies": {
    "@eslint/eslintrc": "^3",
    "@stylexjs/eslint-plugin": "^0.10.1",
    "@stylexswc/rs-compiler": "^0.6.5",
    "@types/node": "^20",
    "@types/react": "^19",
    "@types/react-dom": "^19",
    "eslint": "^9",
    "eslint-config-next": "15.1.7",
    "rimraf": "^6.0.1",
    "typescript": "^5"
  }
}
```

#### next.config.ts
```ts
import path from 'path';
import stylexPlugin from '@stylexswc/nextjs-plugin';
const rootDir = __dirname;

module.exports = stylexPlugin({
  rsOptions: {
    aliases: {
      '@/*': [
        path.join(rootDir, '*'),
      ],
    },
    unstable_moduleResolution: {
      type: 'commonJS',
    },
  }
})({});
```

- Run the development server (e.g., using `next dev`) and observe the warning on the initial page load.

## Expected Behavior

The warning should not be displayed, or an option/suggestion should be provided to configure the `from` option in PostCSS to prevent the warning from appearing on startup.

### Additional Information

- No custom PostCSS configuration file or extra plugins are used.
- The warning suggests setting the `from` option to the CSS file path or to `undefined`.
- The functionality and build process have no issues aside from this startup warning.

Thank you for your support!

**Comment by Dwlad90** [comment]
Hi @yasuhiro-yamamoto,

I created a mini-application according to your instructions and there are no warnings in the console. Also, in any steps there is no use of `@stylexswc\postcss-plugin`.

Here is my demo application launch log:
```bash
next dev
   ▲ Next.js 15.1.7
   - Local:        http://localhost:3000
   - Network:      http://192.168.1.107:3000

 ✓ Starting...
 ✓ Ready in 1521ms
(node:64620) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
 ○ Compiling / ...
 ✓ Compiled / in 1016ms (674 modules)
 GET / 200 in 1193ms
```

Can you clarify how to reproduce this warning?

**Comment by yasuhiro-yamamoto** [comment]
@Dwlad90 

Thank you for taking the time to review this. 
I have created a repository for a mini application that displays a warning in the console.

You can check it out at:
https://github.com/yasuhiro-yamamoto/nextjs-stylex-example

It appears that the warning is triggered when code that creates styles using `stylex.create` is included.
I would appreciate it if you could take a look and share your feedback.

**Comment by Dwlad90** [comment]
@yasuhiro-yamamoto,

Thanks for providing additional information. I was able to fix  `@stylexswc/nextjs-plugin` according to the warning recommendation in #238 . Thank you for your attention to this.

The bugfix is ​​available in version `0.6.6-rc.1`.
